JsonSchema.Generation/SchemaGeneratorContext.cs

```diff
@@ -40,12 +126,17 @@ public ContextCount(SchemaGeneratorContext context)
 		/// The generator configuration.
 		/// </summary>
 		public SchemaGeneratorConfiguration Configuration { get; }
+		/// <summary>
+		/// <see cref="IComparer{MemberInfo}"/> for ordering members when generating a schema with <see cref="PropertyOrder.AsDeclared"/> ordering enabled.
+		/// </summary>
+		internal Lazy<IComparer<MemberInfo>> AsDeclaredMemberInfoComparer { get; }
```

Comment:
A [poor-man's lazy](https://stackoverflow.com/a/63085705/878701) here instead (and a simpler name), please.  This doesn't expose the lazy nature, and we can just use the value we need.  There's also a bit of overhead for `Lazy<T>` that we can avoid.  I'm not worried about thread safety here since this object should never see multiple threads.

```suggestion
		private IComparer<MemberInfo> _memberInfoComparer;
		internal IComparer<MemberInfo> AsDeclaredMemberInfoComparer => _memberInfoComparer ??= new MemberInfoMetadataTokenComparer(Type);
```

Reply:
Internal property name is now `DeclarationOrderComparer`